FluidSynth/FluidSynth.cpp

```diff
@@ -147,7 +226,43 @@ CK_DLL_QUERY( fluidsynth )
     QUERY->add_arg(QUERY, "int", "bankNum");
     QUERY->add_arg(QUERY, "int", "chan");
 
-    fluidsynth_data_offset = QUERY->add_mvar(QUERY, "int", "@f_data", false);
```

Comment:
Im confused why this line was removed / how this chugin works without this line? This is needed to allocate a space in the Chuck_Object for the corresponding instance of the FluidSynth class.